### PR TITLE
Styled the Bookstore App

### PR DESCRIPTION
**Pull request by richie1988**
### Bookstore Styling

- Styled the bookstore App
- Matched the Design And followed the JXS elements
- Implemented the styling guard

### CODE REVIEW

File: src/components/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  width: 100%;
+  height: 92px;
+  background-color: rgb(236, 236, 245);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  box-shadow: 2px 2px 12px rgb(109, 107, 107);
+}
+
```
Comment (algerina) [inline src/components/Navigation.css]: - [ ] Well done styling the app however you should follow the Zeplin design (Same fonts, colors, spacing) and its not the case for the Navbar. Kindly change the colors the font and spacing to match the design.

File: src/components/bookForm.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.book-input button {
+  font-size: 16px;
+  font-weight: bold;
+  padding: 11px;
+  border-radius: 6px;
+  width: 30%;
+}
+
```
Comment (algerina) [inline src/components/bookForm.css:36]: - [ ] Kindly use the same colors and fonts as the Zeplin design for the button and the whole form.

File: src/components/books.css [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+.book-progress {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-left: auto;
+  justify-content: space-between;
+  font-size: 22px;
+}
```
Comment (algerina) [inline src/components/books.css:82]: - [ ] Well done styling the App however your design should match the Zeplin design. Kindly change the colors, spacing and fonts to match the design.

Your design:

Screenshot 2023-10-06 at 6 16 43 PM


Zeplin design:


Screenshot 2023-10-06 at 6 18 10 PM

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @richie1988 ,

Great job so far! However you have some minor issues to work on.

May I highlight the following: 
-  Well done working on the styling✔
- Your code is well organized✔️



## CHANGES REQUIRED:muscle::clap:


Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: src/components/Navigation.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
+.nav-container {
+  width: 100%;
+  height: 92px;
+  background-color: rgb(236, 236, 245);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  box-shadow: 2px 2px 12px rgb(109, 107, 107);
```
Comment (tufoinnkuo10) [inline src/components/Navigation.css]: - [ ] wonderful job so far. please like the previous reviewer mentioned, we are to follow the zeplin design as much a possible. kindly use the original color `#0090ff` for the update buttons and also the add book button has a thesame color as well. nice work. 
yours
Screenshot_39
original
Screenshot_37

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @richie1988 

Great job making some of the changes. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by levy002 — APPROVED** [review]
## Hello @richie1988 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**